lib/models/logger.dart

```diff
@@ -25,6 +25,24 @@ Future<File> getLogsPath() async {
   if (kIsMacOS) {
     dir = path.join((await getLibraryDirectory()).path, "Logs");
   }
+  
+  if (kIsLinux) {
+    // path_provider seems does not support XDG_STATE_HOME,
+    // which is the specification to store application logs on Linux.
+    // See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+    String getXdgStateHome() {
```

Comment:
What's the point of creating a function inside a function if it's only used once inside? It could br created as a private method in the class.

Reply:
@KRTirtho I have rebased changes to branch `dev` and used private function instead nested function.